docs/content/doc/usage/agit-support.en-us.md

```diff
@@ -0,0 +1,45 @@
+---
+date: "	2022-09-01T20:50:42+0000"
+title: "Usage: Agit Setup"
+slug: "agit-setup"
+weight: 12
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Agit Setup"
+    weight: 12
+    identifier: "agit-setup"
+---
+
+# Agit Setup
+
+In Gitea `1.13`, support for [agit](https://git-repo.info/en/2020/03/agit-flow-and-git-repo/) was added.
+
+## Creating PRs with Agit
+
+Agit allows to create PRs while pushing code to the remote repo. \
+This can be done by pushing to the branch followed by a specific refspec (a location identifier known to git). \
+The following example illustrates this:
+
+```shell
+git push origin HEAD:refs/for/master
+```
+
+The command has the following structure:
+
+- `HEAD`: The target branch
+- `refs/<for|drafts|for-review>/<branch>`: The target PR type
+  - `for`: Create a normal PR with `<branch>` as the target branch
+- `<branch>/<session>`: The target branch to open the PR
+- `-o <topic|title|description>`: Options for the PR
+  - `title`: The PR title
+  - `topic`: The PR topic
+  - `description`: The PR description (in Markdown format)
```

Comment:
![image](https://user-images.githubusercontent.com/51889757/188208209-96de64bc-4759-4ac7-aa50-9b17716c2deb.png)
1. The "topic" seems to be the branch name (I set `-o topic="Branchname"`)
2. That doesn't seem like Markdown is supported?
3. The current implementation appears to be absolutely buggy:
  - The base branch exists nowhere
    - how do I know if such refspecs exist? The UI doesn't show them **at all**. There could be a million of them without me knowing. 
    - Clicking on the branch results (understandably?) in a 404 
![image](https://user-images.githubusercontent.com/51889757/188212398-129c3b37-53de-43a6-aa40-2ea0f88f09b4.png)
  - Git never complains, even when an invalid name was pushed (to be fair, I don't know if we can or should fix that at all)
```suggestion
  - `topic`: The branch name the PR should be opened for 
  - `description`: The PR description
```